xpring4j.iml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it necessary to add this file to version control? Generally I would advocate for not checking IDE settings in.

Reply:
Ugh sorry, this should be git-ignored after the changes in the dependent PR too. Removed.